ods_ci/utils/scripts/SplitSuite.py

```diff
@@ -18,7 +18,7 @@ def chunked(lst, nchunk):
 
 
 def fetch_subsuites(parent):
-    children = [child for child in parent.suites]
+    children = set(parent.suites)
```

Comment:
hm could you use list instead? I think there is technically the possibility to have suites with same name (under different subfolders). It shouldn't happen, but it could.

Reply:
I don't remember why I put set here, will use list instead.

Reply:
thanks!